pkg/controllers/system_jobs.go

```diff
@@ -34,27 +34,31 @@ func (job SystemJob) Run() {
 		Depth:     10,
 		Namespace: job.Namespace,
 	}
-	systems := topology.Run(opts, job.SystemTemplate)
-	for _, system := range systems {
-		system.Name = job.SystemTemplate.Name
-		system.Namespace = job.SystemTemplate.Namespace
-		system.Labels = job.SystemTemplate.Labels
-		systemTemplateID, _ := uuid.Parse(job.SystemTemplate.GetPersistedID())
-		system.SystemTemplateID = &systemTemplateID
-		systemID, comps, err := db.PersistSystem(system)
-		if err != nil {
-			logger.Errorf("error persisting the system: %v", err)
-		}
-		dbComps, err := db.GetActiveComponentsWithSystemID(systemID)
+	components := topology.Run(opts, job.SystemTemplate)
+	systemTemplateID, err := uuid.Parse(job.SystemTemplate.GetPersistedID())
+	if err != nil {
+		logger.Errorf("error finding the systemTemplateID")
```

Comment:
When wouldn't we have a persistent id ?

Reply:
I don't see a scenario where this would execute but in case there is some scenario we would want to log it? and handle it accordingly?

The other alternative would be to use uuid.MustParse which would panic in case of an error.... or to just through the error variable all together